src/main/scala/Countdown.scala

```diff
@@ -0,0 +1,77 @@
+object Countdown extends App {
+  import scala.io.Source
+
+  def vowel(): String = {
+    val letterID = scala.util.Random.nextInt(4)
+    letterID match {
+      case 0 => "a"
+      case 1 => "e"
+      case 2 => "i"
+      case 3 => "o"
+      case 4 => "u"
+    }
+  }
+
+  def consonant():String= {
+    val letterID = scala.util.Random.nextInt(20)
```

Comment:
('a' to 'z').toArray(arrayOfLetters(Random.nextInt(20))) would do the same thing you're doing with like 30 lines of code, the only difference is that it returns a character so you would need to convert it to string